webapp/app.py

```diff
@@ -34,15 +40,58 @@
 @app.errorhandler(401)
 @app.errorhandler(403)
 @app.errorhandler(404)
-def error_handler(exception=None):
-    code = getattr(exception, "code")
-    return {"status": f"{code}", "message": str(exception)}, code
+def error_handler(error=None):
+    code = getattr(error, "code")
+    return {"status": code, "message": str(error)}, code
```

Comment:
If `code` (which is presumably an `int`) gets converted to a `str` automatically for passing to the context, why doesn't `error`?

Reply:
`code` is not converted to a `str`, it shows in the JSON payload as a numeric type.

Reply:
Oh I see! Fair enough.